src/theme/theme.ts

```diff
@@ -258,6 +269,15 @@ const themeCustomized = createTheme({
         }),
       },
     },
+    MuiContainer: {
+      styleOverrides: {
+        root: () => ({
+          [themeBase.breakpoints.up('lg' as Breakpoint)]: {
```

Comment:
This will be applied globally, are you sure that all pages has been checked? (just verification)